fineract-provider/src/test/java/org/apache/fineract/mix/report/service/XBRLBuilderTest.java

```diff
@@ -79,13 +79,10 @@ public void shouldCorrectlyBuildMap() {
             nodes = DocumentBuilderFactory.newInstance().newDocumentBuilder().parse(new ByteArrayInputStream(result.getBytes()))
                     .getElementsByTagName("Assets");
         } catch (final SAXException e) {
-            // TODO Auto-generated catch block
```

Comment:
This would ideally be raised as a separate PR (because it has nothing to do with the other changes proposed here), but it's so minor that it doesn't matter/ it's OK for me.

Reply:
True, I'll make a new PR for that

Reply:
#663 did this, but in resolving the conflicts it looks like somehow this came back?